cypress/e2e/2-advanced-examples/assertions.spec.js

```diff
@@ -51,6 +51,7 @@ context("Assertions", () => {
     // https://on.cypress.io/assertions
     it("expect - make an assertion about a specified subject", () => {
       // We can use Chai's BDD style assertions
+      // eslint-disable-next-line no-unused-expressions
```

Comment:
linting complaining about the mock files that come with the upgrade. So linting works!